internal/pkg/cli/svc_deploy.go

```diff
@@ -321,6 +321,27 @@ func (o *deploySvcOpts) configureContainerImage() error {
 	return nil
 }
 
+func (o *deploySvcOpts) getTopics(unmarshaledManifest interface{}) (map[string]string, error) {
```

Comment:
The grand vision 🔮 is to keep the commands in the `cli` pkg to be as lean as possible.    
I know that we don't do that well today 😞 but ideally we should try to move to a state where all commands only make calls to subpackages without any transformation logic here.

Could we move this logic to a lower level package like `stack` and `template`?

For example,
```go
// In package stack.
type RuntimeConfig struct {
    ...
    AccountID string
    Region string
}


// In package template.
type Topic struct {
   Name string
   
   region      string
   accountID   string
   app         string
   env         string
}
func NewTopic(region, account, app, env string) Topic

func (t topic) ARN() string { ... }

type Publish struct {
   Topics []Topic
}
```

Reply:
Adjusted packages and moved integration into svc/job and related tests to another branch